test_runner/regress/test_wal_acceptor.py

```diff
@@ -2606,10 +2606,10 @@ def all_evicted():
         assert n_evicted  # make mypy happy
         assert int(n_evicted) == n_timelines
 
-    wait_until(60, 0.5, all_evicted)
+    wait_until(all_evicted)
```

Comment:
This replaces a 30 second timeout with a 20 second timeout  (this test failed with a timeout in this PR's CI: https://neon-github-public-dev.s3.amazonaws.com/reports/pr-9936/12085308134/index.html#/testresult/8acc4ba7a0987d2b)